app/src/main/res/layout/cover_fragment.xml

```diff
@@ -51,11 +59,120 @@
             android:layout_height="wrap_content"
             android:ellipsize="none"
             android:gravity="center_horizontal"
-            android:textSize="@dimen/text_size_small"
             android:maxLines="2"
             android:textColor="?android:attr/textColorPrimary"
             android:textIsSelectable="true"
+            android:textSize="@dimen/text_size_small"
             tools:text="Episode" />
+
+        <Space
+            android:id="@+id/vertical_divider"
+            android:layout_width="match_parent"
+            android:layout_height="8dp"
+            android:visibility="gone" />
+
+    </LinearLayout>
+
+    <Space
+        android:id="@+id/details_spacer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_weight="1" />
+
+    <LinearLayout
+        android:id="@+id/episode_details"
+        android:layout_width="wrap_content"
+        android:layout_height="36dp"
+        android:layout_weight="0"
+        android:baselineAligned="false"
+        android:orientation="horizontal"
+        android:paddingLeft="8dp"
+        android:paddingRight="8dp">
+
+        <LinearLayout
+            android:id="@+id/openDescriptionButton"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginLeft="8dp"
+            android:layout_marginEnd="8dp"
+            android:layout_marginRight="8dp"
+            android:background="@drawable/grey_border"
+            android:clickable="true"
+            android:focusable="true"
+            android:gravity="center"
+            android:minWidth="150dp"
+            android:layout_weight="1"
+            android:orientation="horizontal">
+
+            <ImageButton
+                android:id="@+id/openDescription"
+                android:layout_width="wrap_content"
+                android:layout_height="36dp"
+                android:background="?attr/selectableItemBackgroundBorderless"
+                android:contentDescription="@string/shownotes_contentdescription"
+                android:padding="2dp"
+                app:srcCompat="@drawable/ic_info" />
+
+            <TextView
+                android:id="@+id/shownotes_label"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="none"
+                android:gravity="center_horizontal"
+                android:maxLines="2"
+                android:paddingLeft="2dp"
+                android:text="@string/shownotes_label"
+                android:textColor="?android:attr/textColorSecondary"
+                android:textSize="@dimen/text_size_navdrawer" />
```

Comment:
I would use a fixed text size here. Reason is that someone who changes the nav drawer text size probably does not expect that it changes the text size here, too.